rds/rds.tf

```diff
@@ -20,28 +20,40 @@ resource "aws_db_instance" "wp_db_main" {
     final_snapshot_identifier   = "${var.service}-wp-${var.environment}-final-db-snapshot"
     backup_window               = var.db_backup_window
     backup_retention_period     = var.db_backup_retention_period
+    snapshot_identifier         = var.db_snapshot_identifier ? data.aws_db_snapshot.latest_snapshot.id : null
+
+    lifecycle {
+        ignore_changes = [
+            "snapshot_identifier"]
+    }
 
     tags = {
-        Name            = "${var.service}-wp-${var.environment}-ma"
-        Service         = var.service
-        Environment     = var.environment
-        CostCentre      = var.cost_centre
-        Owner           = var.owner
-        CreatedBy       = var.created_by
-        Terraform       = true
+        Name        = "${var.service}-wp-${var.environment}-ma"
```

Comment:
Did you mean to leave this name unmodified? It's different from the actual `identifier` above?

Reply:
I don't think it matters, but I'll change it.